### DISCUSSION TITLE
How to create a slider with touch events while having different mouse events on the same component. React JS

### DISCUSSION THREAD

**Discussion by RaulRogojan** [community/community#38349] [Other Feature Feedback, Questions, & Ideas]
Hi, 
I am trying to build a "fancy" nav menu. So far I have build the desktop version, where the "track" moves based on the mouse position. This works almost great but I have 2 issues. 

1. I have a bug where if I click on any nav link the navigation animation will glitch and the click event will not happen. 
2. When trying to build the mobile version using touch events, as I want this behavior to be specific for mobile only, I encounter some issues mainly with the touch move event. 

Here is the component and the CSS for it.
https://github.com/CodyHiII/files

Thank you

**Comment by ossanna16** [comment]
Hi @CodyHiII,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on ReactJS. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
